crates/cxx-qt-gen/update_expected.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+
+set -ex
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
interesting way of doing it, i usually do this

```bash
SCRIPT=$(realpath "$0")
SCRIPTPATH=$(dirname "$SCRIPT")
```

Reply:
Found this on Stackoverflow :man_shrugging: 
Doesn't really matter I guess...

Reply:
Just seems super complicated in comparison and we've already used the realpath + dirname route elsewhere :thinking:

Reply:
Or `SCRIPT_DIR=$(dirname "$(realpath "$0")")` also works for a one liner, but you get the double quote fun

Reply:
Done